mods/d2k/maps/harkonnen-06b/harkonnen06b.lua

```diff
@@ -0,0 +1,200 @@
+
+OrdosReinforcements =
+{
+	easy =
+	{
+		{ "combat_tank_o", "light_inf", "raider" },
+		{ "raider", "trooper" },
+		{ "quad", "trooper", "trooper", "combat_tank_o"},
+		{ "siege_tank", "quad" },
+		{ "combat_tank_o", "combat_tank_o", "light_inf", "light_inf", "light_inf", "light_inf" },
+		{ "trooper", "trooper", "trooper", "trooper", "trooper" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "missile_tank" }
+	},
+
+	normal =
+	{
+		{ "combat_tank_o", "raider", "raider" },
+		{ "raider", "raider" },
+		{ "quad", "trooper", "trooper", "trooper", "combat_tank_o"},
+		{ "raider", "raider" },
+		{ "siege_tank", "combat_tank_o" },
+		{ "trooper", "trooper", "trooper", "trooper", "trooper", "trooper" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "missile_tank" },
+		{ "combat_tank_o", "combat_tank_o", "siege_tank" }
+	},
+
+	hard =
+	{
+		{ "combat_tank_o", "combat_tank_o", "raider" },
+		{ "raider", "raider", "trooper" },
+		{ "quad", "trooper", "trooper", "trooper", "trooper", "combat_tank_o"},
+		{ "raider", "raider", "light_inf" },
+		{ "siege_tank", "combat_tank_o", "quad" },
+		{ "trooper", "trooper", "trooper", "trooper", "trooper", "trooper", "trooper" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "missile_tank" },
+		{ "combat_tank_o", "combat_tank_o", "siege_tank", "siege_tank" },
+		{ "missile_tank", "quad", "quad", "raider", "raider" }
+	}
+}
+
+OrdosStarportReinforcements =
+{
+	easy = { "raider", "missile_tank", "combat_tank_o", "quad", "deviator", "deviator" },
+	normal = { "raider", "missile_tank", "missile_tank", "quad", "deviator", "deviator" },
+	hard = { "raider", "raider", "missile_tank", "missile_tank", "quad", "quad", "deviator", "deviator" }
+}
+
+OrdosAttackDelay =
+{
+	easy = DateTime.Minutes(3),
+	normal = DateTime.Minutes(2) + DateTime.Seconds(20),
+	hard = DateTime.Minutes(1)
+}
+
+OrdosStarportDelay =
+{
+	easy = DateTime.Minutes(7),
+	normal = DateTime.Minutes(6),
+	hard = DateTime.Minutes(5)
+}
+
+OrdosAttackWaves =
+{
+	easy = 7,
+	normal = 8,
+	hard = 9
+}
+
+InitialOrdosReinforcements =
+{ 
+	{ "trooper", "trooper", "trooper", "trooper", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf" },
+	{ "combat_tank_o", "combat_tank_o", "quad", "quad", "raider", "raider" }
+}
+
+OrdosPaths =
+{
+	{ OrdosEntry1.Location, OrdosRally1.Location },
+	{ OrdosEntry2.Location, OrdosRally2.Location },
+	{ OrdosEntry3.Location, OrdosRally3.Location },
+	{ OrdosEntry4.Location, OrdosRally4.Location },
+	{ OrdosEntry5.Location, OrdosRally5.Location },
+	{ OrdosEntry6.Location, OrdosRally6.Location }
+}
+
+InitialOrdosPaths =
+{
+	{ OrdosEntry7.Location, OrdosRally7.Location },
+	{ OrdosEntry8.Location, OrdosRally8.Location },
+	{ OrdosEntry9.Location, OrdosRally9.Location }
+}
+
+OrdosInterval =
```

Comment:
Maybe I overlook something but what is `OrdosInterval` used for?

Reply:
`FremenInterval` was used by Sietchs producing fremen on All4 and Har4, it is not used in this, or other maps, looks like i just renamed and forget about it. I think 6a is merged with that, i'll remove it from both.